android/app/src/main/res/layout-land/onmap_downloader.xml

```diff
@@ -45,14 +45,20 @@
         android:gravity="center_horizontal"
         tools:text="Some name very loooooooooong name"
         tools:ignore="UnusedAttribute"/>
+
       <TextView
-        android:id="@+id/downloader_size"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_margin="@dimen/margin_half"
-        android:textAppearance="@style/MwmTextAppearance.Body2"
-        android:gravity="center_horizontal"
-        tools:text="1000 MB"/>
+          android:id="@+id/downloader_size"
+          android:layout_width="match_parent"
+          android:layout_height="24sp"
+          android:layout_margin="@dimen/margin_half"
+          android:gravity="center_horizontal|center_vertical"
+          android:lines="1"
+          android:textAppearance="@style/MwmTextAppearance.Body2"
+          app:autoSizeMaxTextSize="16sp"
```

Comment:
Was the previous size taken from some style, so changing it in one place will scale it everywhere else automatically?

Reply:
`16sp` was the previous size set in that style. But I can set it to `@dimen/text_size_body_2` which is where that's coming from. Do we want a constant for min as well?

Reply:
If it's used only here, there's no need for a minimum constant. If there's a similar usage somewhere else, it would be better to reuse a constant.